### PR TITLE
Log in, log out, remember me

### PR DESCRIPTION
**Pull request by agalkiewicz**

### CODE REVIEW

File: app/assets/javascripts/sessions.coffee [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (mmarcinkowski) [inline app/assets/javascripts/sessions.coffee]: do we need this empty file ?

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user && user.authenticate(params[:session][:password])
```
Comment (mmarcinkowski) [inline app/controllers/sessions_controller.rb]: maybe `params.fetch(:session)[:key]` ? it provides a better error ;) please read about it.

Comment (agalkiewicz) [inline app/controllers/sessions_controller.rb]: Corrected.

File: app/assets/javascripts/sessions.coffee (hunk 1 above)
Comment (agalkiewicz) [inline app/assets/javascripts/sessions.coffee]: Deleted.

**Pull request merged**